addons/l10n_ar/models/account_journal.py

```diff
@@ -56,23 +57,19 @@ def _get_journal_letter(self, counterpart_partner=False):
                 '9': ['E'],
                 '10': ['E'],
                 '13': ['B', 'C', 'I'],
+                '99': ['B', 'C', 'I']
             },
         }
         if not self.company_id.l10n_ar_afip_responsibility_type_id:
             action = self.env.ref('base.action_res_company_form')
             msg = _('Can not create chart of account until you configure your company AFIP Responsibility and VAT.')
             raise RedirectWarning(msg, action.id, _('Go to Companies'))
 
-        letters = letters_data['issued' if self.type == 'sale' else 'received'][
-            self.company_id.l10n_ar_afip_responsibility_type_id.code]
-        if not counterpart_partner:
-            return letters
-
-        if not counterpart_partner.l10n_ar_afip_responsibility_type_id:
-            letters = []
-        else:
-            counterpart_letters = letters_data['issued' if self.type == 'purchase' else 'received'][
-                counterpart_partner.l10n_ar_afip_responsibility_type_id.code]
+        letters = letters_data['issued' if self.type == 'sale' else 'received'].get(
+            self.company_id.l10n_ar_afip_responsibility_type_id.code, [])
```

Comment:
.get seems unnecessary

Reply:
The user should not change AFIP responsibilities...  And I don't see why we should not provide all the combinations then.

Reply:
Okay, maybe we should put a warning if the user tries to change the responsibility of the company.. and about the combinations, the remaining ones are responsibilities no much used that are not implemented yet. I will change this, but maybe we can talk about these points better in the meeting.

Reply:
@jco-odoo Done!